pkg/api/v1/types/types.go

```diff
@@ -0,0 +1,88 @@
+package types
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/google/uuid"
+	ptypes "github.com/metal-toolbox/conditionorc/pkg/types"
+	"github.com/pkg/errors"
+)
+
+type ServerResponse struct {
+	StatusCode int                 `json:"statusCode,omitempty"`
+	Message    string              `json:"message,omitempty"`
+	Record     *ConditionResponse  `json:"record,omitempty"`
```

Comment:
feels redundant to have a singular and multiple fields that are basically the same thing. Also the difference between `ConditionResponse` and `ConditionsResponse` is hard to discern and a code-maintenance drag. Consider only allowing the multiple-response type and explicitly allowing `len(Records)>=0`.